<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWidget</string>
  </property>
  <widget class="QLabel" name="pictureLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureInfoLabel">
   <property name="geometry">
    <rect>
     <x>805</x>
     <y>540</y>
     <width>291</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>805</x>
     <y>0</y>
     <width>291</width>
     <height>250</height>
    </rect>
   </property>
   <property name="title">
    <string>连接设置</string>
   </property>
   <widget class="QPushButton" name="connectCameraForUDPPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>连接相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchCameraForTCPPushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索相机(TCP)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverIPLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="udpClosePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>断开UDP服务器</string>
    </property>
   </widget>
   <widget class="QComboBox" name="udpCamerasComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;地址：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tcpCamerasComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="serialPortPushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectCameraForTCPPushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>连接相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="udpClientConnectPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>连接UDP服务器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tcpClientConnectPushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>连接TCP服务器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tcpClosePushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>断开TCP服务器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;端口：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialPortComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverPortLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchCameraforUDPPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索相机(UDP)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>805</x>
     <y>250</y>
     <width>291</width>
     <height>250</height>
    </rect>
   </property>
   <property name="title">
    <string>其他设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
